src/index.css

```
@@ -5,6 +5,33 @@
+  .dark {
+    --surface-base: 3 7 18;
+    --surface-card: 17 24 39;
+    --surface-glass: 255 255 255;
+    --surface-overlay: 10 15 26;
+    --line-default: 16 19 26;
+    --line-glass: 255 255 255;
+    --input-bg: 10 15 26;
+    --input-border: 55 65 81;
+    --input-text: 249 250 251;
+    --input-placeholder: 156 163 175;
+    --shadow-card: 0 14px 32px rgba(0, 0, 0, 0.35);
```

Stylelint (17.2.0)

[error] Expected "0.35" to be "35%" (alpha-value-notation)

(alpha-value-notation)